serve/tests/unittests/triton/test_builder.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+from pathlib import Path
+import filecmp
+from tempfile import TemporaryDirectory
+
+from serve.triton.builder import Builder
+from . import EXPECTED_FILE_PATH
+
+
+class TestBuilder(unittest.TestCase):
+    '''
+    Test triton buidler.
+    '''
+    def test_builder(self):
+        test_dag = {
+            'start': {
+                'op_name': 'dummy_input', 'init_args': None, 'child_ids': ['cb2876f3']
+            },
+            'cb2876f3': {
+                'op_name': 'local/triton_py', 'init_args': {}, 'child_ids': ['fae9ba13']
+            },
+            'fae9ba13': {
+                'op_name': 'local/triton_nnop', 'init_args': {'model_name': 'test'},'child_ids': ['end']
+            },
+            'end': {
+                'op_name': 'end', 'init_args': None, 'call_args': None, 'child_ids': []
+            }
+        }
+        with TemporaryDirectory(dir='./') as root:
+            builer = Builder(test_dag, root, ['tensorrt'])
+            assert builer.load() is True
+            self.assertTrue(builer.build())
+
```

Comment:
consider providing a function `build_triton_models` wrapping Builder.__init__(), load(), and build()